app/src/main/java/io/github/boogiemonster1o1/eyeyoureadyforit/data/Leaderboard.java

```diff
@@ -0,0 +1,42 @@
+package io.github.boogiemonster1o1.eyeyoureadyforit.data;
```

Comment:
this should probably go in `i.g.b.e.data.stats` package

Reply:
along with anything else related to stats in `i.g.b.e.data`

Reply:
good idea
is `DataDao` included though since its purpose is to have db queries in general and not just for stats
however pretty much its entire contents is for stats alone

Reply:
it can be used for other purposes in the future such as settings. best to leave it where its at